tests/Integration/Behaviour/Features/Scenario/Tax/tax_management.feature

```diff
@@ -0,0 +1,56 @@
+@reset-database-before-feature
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s tax
+Feature: Manage tax
+  As a employee
+  I must be able to correctly add, edit and delete tax
+
+  Scenario: Adding new tax
+    When I add new tax "tax-1" with following properties:
+      | name         | my custom tax 500 |
+      | rate         | 0.5               |
+      | is_enabled   | 1                 |
+    Then tax "tax-1" name in default language should be "my custom tax 500"
+    And tax "tax-1" rate should be 0.500
+    And tax "tax-1" should be enabled
+
+  Scenario: Editing tax
+    When I edit tax "tax-1" with following properties:
+      | name         | my custom tax 300 |
+      | rate         | 0.15              |
+      | is_enabled   | 0                 |
```

Comment:
If you use `PrimitiveUtils::castStringBooleanIntoBoolean()` on this table row, you can write

```suggestion
      | is_enabled   | 0                 |
```
which is easier to read